subprojects/docs/src/docs/userguide/compatibility.adoc

```diff
@@ -24,10 +24,10 @@ Java 6 and 7 can still be used for <<building_java_projects.adoc#sec:java_cross_
 Any supported version of Java can be used for compile or test.
 
 == Kotlin
-Gradle is tested with Kotlin 1.3.21 through 1.4.20.
+Gradle is tested with Kotlin 1.3.72 through 1.4.20.
 
 == Groovy
 Gradle is tested with Groovy 1.5.8 through 2.5.12.
 
 == Android
-Gradle is tested with Android Gradle Plugin 3.4, 3.5, 3.6, 4.0, 4.1 and 4.2. Alpha and beta versions may or may not work.
```

Comment:
@eskatos I saw Sterling already removed 3.x on `master` because of the deprecation warnings, so I changed the doc accordingly.